lib/account.rb

```diff
@@ -0,0 +1,51 @@
+# Update the Account class to be able to handle all of these fields from the CSV file used as input.
+# For example, manually choose the data from the first line of the CSV file and ensure you can create a new instance of your Account using that data
+
+require 'csv'
+
+
+module Bank
+  class Account
+    attr_accessor :id, :balance, :date_created
```

Comment:
Should these be `attr_accessor`? Seems to me that `attr_reader` would be more appropriate - you don't want people trying to modify `@balance` directly, they should go through `withdraw` or `deposit` instead.